charts/spark-operator-chart/values.yaml

```
@@ -409,6 +409,31 @@ spark:
     # -- Optional annotations for the spark application RBAC resources.
     annotations: {}
 
+  # Kerberos configuration for Spark applications
+  kerberos:
+    # -- Enable Kerberos authentication support for Spark applications.
+    enabled: false
```

```suggestion
    enable: false
```

Use `kerberos.enable` to be consistent with other occurrences, e.g. `controller.leaderElection.enable` and `webhook.enable`.